vector/src/main/res/values-fr/strings.xml

```diff
@@ -239,51 +229,22 @@
     <string name="dialog_title_confirmation">Confirmation</string>
     <string name="dialog_title_warning">Attention</string>
     <string name="bottom_action_favourites">Favoris</string>
-<<<<<<< HEAD
-    <string name="bottom_action_people">Personnes</string>
-    <string name="bottom_action_rooms">Conversations</string>
-    <string name="home_filter_placeholder_home">Rechercher des conversations</string>
-    <string name="home_filter_placeholder_favorites">Rechercher des favoris</string>
-    <string name="home_filter_placeholder_people">Rechercher des contacts</string>
-    <string name="home_filter_placeholder_rooms">Rechercher des conversations</string>
-    <string name="invitations_header">Invitations</string>
-    <string name="low_priority_header">Priorité basse</string>
-    <string name="direct_chats_header">Discussions</string>
-    <string name="template_local_address_book_header">Contacts ${app_name}</string>
```

Comment:
Some of these strings have probably been changed for Tchap, please can you verify the wordings in the git history? Especially for "Personnes / Participants" and "Conversations / Salons"

Reply:
Same remark for next strings conflicts

Reply:
i added  comments: "Tchap: custom string" before each string updated for tchap.